tardis/adapters/sites/kubernetes.py

```diff
@@ -71,6 +72,11 @@ def hpa_client(self) -> k8s_client.AutoscalingV1Api:
     async def deploy_resource(
         self, resource_attributes: AttributeDict
     ) -> AttributeDict:
+        drone_environment = self.drone_environment(
+            resource_attributes.drone_uuid,
+            resource_attributes.obs_machine_meta_data_translation_mapping,
+        )
```

Comment:
From looking at the code here I don't get the point of having this `drone_environment`. Is it to better distinguish the different drones?

Reply:
All site adapter except OpenStack and CloudStack set this environment variables inside the drone. So that the HTCondor OBS can use `TardisDroneUuid` as `STARTD_NAME`, which used by TARDIS to match drones to machines in HTCondor. In addition, `TardisDroneCores` and `TardisDroneMemory` can be used to limit the amount CPU cores and Memory advertised by HTCondor. Otherwise HTCondor is advertising the entirely available Cores and Memory of the machine. 

Example HTCondor config snippet:
```
# Use TardisDroneUuid to identify drones in OBS
STARTD_NAME = $ENV(TardisDroneUuid)

# Limit CPU and Memory to the capacity of the drone
NUM_CPUS = $ENV(TardisDroneCores)
MEMORY = $ENV(TardisDroneMemory)
```